cmd/csi-node-driver-registrar/node_register.go

```diff
@@ -36,6 +37,15 @@ func nodeRegister(
 	// pluginswatcher infrastructure. Node labeling is done by kubelet's csi code.
 	registrar := newRegistrationServer(csiDriverName, *kubeletRegistrationPath, supportedVersions)
 	socketPath := fmt.Sprintf("/registration/%s-reg.sock", csiDriverName)
+
+	ticker := time.NewTicker(5 * time.Second)
+	for {
+		<-ticker.C
+		go periodicallyCheck(socketPath, registrar)
```

Comment:
Do we need to spawn a new goroutine? Is there ever a case where reconciling the socket could take a long time?